en/docs/graphql-api.md

```diff
@@ -0,0 +1,79 @@
+# GraphQL APIs
+
+Explore how you can easily design, develop, and manage GraphQL APIs in Choreo to implement your business API strategy.
+
+!!! tip
+    GraphQL API is currently available in Choreo as a preview feature. For instructions to enable it, see [Preview Features](references/preview-features.md).
```

Comment:
We need to change this to a `Note` as it is a notice for the users to understand this is a preview version.

Example: https://docs.github.com/en/get-started/using-github/github-command-palette
![image](https://user-images.githubusercontent.com/8376040/185418221-a9a5ee82-4f6e-4316-8db2-6475c9cb015c.png)

Reply:
If possible, shall we put feature enable details inside the `Note`?

Reply:
Added the complete details, but made the note an expandable because the images used take space and the user would not see the key content without scrolling.